app/javascript/mastodon/features/ui/index.js

```diff
@@ -176,8 +181,13 @@ export default class UI extends React.PureComponent {
     const { width, draggingOver } = this.state;
     const { children } = this.props;
 
+    const classNames = ['ui'];
+
+    if (this.props.systemFontUi)
+      classNames.push('system-font');
+
     return (
-      <div className='ui' ref={this.setRef}>
+      <div className={classNames.join(' ')} ref={this.setRef}>
```

Comment:
@unarist I didn't know about that! I'll update the code to use it for consistency.